README.md

```diff
@@ -83,6 +84,9 @@ format>
 
 Where:
 
+--allow-v2-api
+Set to allow usage of the v2 api. (Not recommended). Default: false
```

Comment:
Can we expand the comment indicating the date of the eventual death of v2 support?